10.9-libcxx/stable/main/finkinfo/sci/nco.info

```diff
@@ -91,21 +92,24 @@ Replaces: <<
 	%N453-dev,
 	%N455-dev,
 	%N466-dev,
-	%N469-dev,
-	%N471-dev
+	%N469-dev
 <<
 
 # Unpack Phase:
 Source: https://github.com/nco/nco/archive/%v.tar.gz
 SourceRename: %n-%v.tar.gz
-Source-MD5: ae7640bb18b61bb86cde93ee70800087
-Source-Checksum: SHA1(070db5e0d4b966b1adc5f29074c3b9324bd540bd)
+Source-MD5: 84296a0ec44b9a58dab198fda9fac7f0
+Source-Checksum: SHA1(e39bdb3d50df0c7bb78c808e5b69907f9d2b3f75)
 
 # Patch Phase:
 PatchScript:  <<
 #!/bin/sh -ev
+	# Make sure to link against newly built libnco.dylib
 	perl -pi -e 's|-lnco|%b/src/nco/.libs/libnco.dylib|' src/nco++/Makefile.in
-	perl -pi -e "s/(apple.*)enable_shared='no'/\1/; s/(10\.\[012\])(\*)/\1\,.\2/" configure
+	# Patch configure to force enable_shared, since it suggests that Darwin cannot create shared linraries
```

Comment:
linraries → libraries

Reply:
That you for pointing out the typo. This is fixed now in the latest commit (52426b2)